fish_passage_fieldwork.Rmd

```diff
@@ -0,0 +1,37 @@
+# Fish Passage Field Work Guide
+
+## Field Work Prep
+
+### Apps and Programs Used
+
+1. [Timestamp Camera](http://www.timestampcamera.com/) is used to take photos (landscape only) in the field. Time and utm settings can be found in Zotero with the citation key `irvine2021FishPassage`.
+
+2. [Avenza Maps](https://store.avenza.com/) is key for loading georeferenced pdf field maps from <https://hillcrestgeo.ca/outgoing/fishpassage/projects/>. We also add `kml` files of our target sites (taken from safety plans) to `Avenza` to make navigation easy. The paid version is necessary to load many maps at the same time for big projects.
```

Comment:
good reminder. you should purchase avenza so you can keep more than 3 backup maps in your phone.  don't forget the invoice when you expense.  Should note that it is an APP for the phone and can get it from the `App Store` (Apple mobile devices) or `Google Play Store` (Android mobile devices)